core/modules/taxonomy/taxonomy.module

```diff
@@ -1206,7 +1206,8 @@ function taxonomy_vocabulary_load($machine_name) {
 }
 
 /**
- * Legacy wrapper to return the parents of a term by TID.
+ * Legacy wrapper to return the vocabulary object matching a vocabulary machine
+ * name.
```

Comment:
The summary for a function should all fit on one line https://api.backdropcms.org/doc-standards#order

I'd suggest removing the second "vocabulary" word to make it fit in 80 characters.

Reply:
I've gone ahead and done this.